polkadot/zombienet-sdk-tests/tests/functional/spam_statement_distribution_requests.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test if parachains progress when group is getting spammed by statement distribution requests.
+
+use anyhow::anyhow;
+
+use crate::helpers::assert_para_throughput;
```

Comment:
You should use it from `cumulus_zombienet_sdk_helpers`, since Alin is removing the helpers from polkadot in #7993 

```suggestion
use cumulus_zombienet_sdk_helpers::assert_para_throughput;
```